microsoft-edge/devtools-guide-chromium/security/index.md

```diff
@@ -57,29 +57,33 @@ If you haven't set up HTTPS on your server, [Let's Encrypt](https://letsencrypt.
 
 ### Mixed content
 
-**Mixed content**<!--[mixed content](/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content)--> means that the main origin of a page is secure, but the page requested resources from non-secure origins.  Mixed content pages are only partially protected because the HTTP content is accessible to sniffers and vulnerable to man-in-the-middle attacks.
+**Mixed content**<!--[mixed content](/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content)--> means that the main origin of a page is secure, but the page requested resources from non-secure origins.  Mixed content pages are only partially protected because the HTTP content is accessible to sniffers and vulnerable to [man-in-the-middle attacks](https://wikipedia.org/wiki/Man-in-the-middle_attack).
 
-![Mixed content](./index-images/security-security-overview-mixed-secure.png)
+![Mixed content](./index-images/overview-mixed-secure.png)
 
-In the previous figure, select **View 1 request in Network panel** to open the **Network** tool and apply the `mixed-content:displayed` filter so that the **Network Log** only shows non-secure resources.
-
-![Mixed resources in the Network Log](./index-images/security-network-filter.png)
+In Microsoft Edge, mixed content requests are automatically upgraded to HTTPS when possible. If the resource is not available over HTTPS, the request is blocked.
 
 
 <!-- ====================================================================== -->
 ## View details
```

Comment:
i consider non-blocking and arguably out of scope for this PR:
* h2 followed by h3 without any sentence in between.
* not applied h2/h4/h6 pattern (d/k whole page outline of headings to see if can apply)